lib/team.rb

```diff
@@ -0,0 +1,8 @@
+class Team
+
+attr_reader  :franchise_id, :team_name, :abbreviation, :link
```

Comment:
remains to be seen whether we will need all these instance variables

Reply:
Agreed